src/block-editor/filters/withBlockBinding.tsx

```diff
@@ -67,7 +67,9 @@ function BoundBlockEdit( props: BoundBlockEditProps ) {
 	return (
 		<>
 			<InspectorControls>
-				<PanelBody title={ __( 'Remote Data', 'remote-data-blocks' ) }>
+				<PanelBody
+					title={ __( sprintf( 'Remote Data Block: %s', remoteDataName ), 'remote-data-blocks' ) }
```

Comment:
This string is not really localizable since it includes a highly variable element

Reply:
Flipping the order around works, though!

```suggestion
					title={ sprintf( __( 'Remote Data Block: %s' ), remoteDataName ), 'remote-data-blocks' ) }
```

Reply:
Whoops, you know what - this was how it was before. I flipped it while going through it as it looked like the wrong order. Good catch